jsonnet/rules.libsonnet

```diff
@@ -301,11 +301,10 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
             // Returns 1 if all control plane nodes are ready and is absent otherwise. Should be used to suppress alerts during control plane upgrades or disruption.
           },
           {
-            expr: 'rate(cluster_monitoring_operator_reconcile_errors_total[15m]) * 100 / rate(cluster_monitoring_operator_reconcile_attempts_total[15m]) > 10',
+            expr: 'max_over_time(cluster_monitoring_operator_last_reconciliation_successful[1h]) == 0',
```

Comment:
`cluster_monitoring_operator_last_reconciliation_successful` this is set to 1 on start, so the alert will fire when CMO is started.

Reply:
Ha, the alert fires because the old instance is still accounted for: we need `max(max_over_time(cluster_monitoring_operator_last_reconciliation_successful[1h])) == 0` instead.

Reply:
thank you, fixing and regenerating :+1: